src/busesApp/controllers.js

```diff
@@ -0,0 +1,58 @@
+import { matchedData } from 'express-validator';
+import handleResponse from '../controllers/handleResponse.js';
+import { Bus } from './models.js';
+import {
+	deleteBus,
+	findBusById,
+	findBusByPlate,
+	findBuses,
+	registerBus,
+	updateBus,
+} from './services.js';
+
+export const postBusHandler = async (req, res) => {
+	const { plateNumber } = req.body;
+	const existingBus = await findBusByPlate(plateNumber);
+	if (existingBus) {
+		return handleResponse(res, 409, { message: res.__('Bus already exists') });
+	}
```

Comment:
you have the same checks in the validation maybe you should choose the one that works better and go with it